datasets/swiss_judgment_prediction/dataset_infos.json

```diff
@@ -1 +1 @@
e": 299910822, "size_in_bytes": 1300293153}, "all+mt": {"description": "\nSwiss-Judgment-Prediction is a multilingual, diachronic dataset of 85K Swiss Federal Supreme Court (FSCS) cases annotated with the respective binarized judgment outcome (approval/dismissal), posing a challenging text classification task. We also provide additional metadata, i.e., the publication year, the legal area and the canton of origin per case, to promote robustness and fairness studies on the critical area of legal NLP.\n", "citation": "@InProceedings{niklaus-etal-2021-swiss,\n  author = {Niklaus, Joel\n                and Chalkidis, Ilias\n                and St\u00fcrmer, Matthias},\n  title = {Swiss-Court-Predict: A Multilingual Legal Judgment Prediction Benchmark},\n  booktitle = {Proceedings of the 2021 Natural Legal Language Processing Workshop},\n  year = {2021},\n  location = {Punta Cana, Dominican Republic},\n}", "homepage": "https://github.com/JoelNiklaus/SwissCourtRulingCorpus", "license": "", "features": {"id": {"dtype": "int32", "id": null, "_type": "Value"}, "year": {"dtype": "int32", "id": null, "_type": "Value"}, "text": {"dtype": "string", "id": null, "_type": "Value"}, "label": {"num_classes": 2, "names": ["dismissal", "approval"], "id": null, "_type": "ClassLabel"}, "language": {"dtype": "string", "id": null, "_type": "Value"}, "region": {"dtype": "string", "id": null, "_type": "Value"}, "canton": {"dtype": "string", "id": null, "_type": "Value"}, "legal area": {"dtype": "string", "id": null, "_type": "Value"}, "source_language": {"dtype": "string", "id": null, "_type": "Value"}}, "post_processed": null, "supervised_keys": null, "task_templates": null, "builder_name": "swiss_judgment_prediction", "config_name": "all+mt", "version": {"version_str": "2.0.0", "description": "", "major": 2, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 834876881, "num_examples": 238818, "dataset_name": "swiss_judgment_prediction"}, "validation": {"name": "validation", "num_bytes": 26209333, "num_examples": 8208, "dataset_name": "swiss_judgment_prediction"}, "test": {"name": "test", "num_bytes": 61849297, "num_examples": 17357, "dataset_name": "swiss_judgment_prediction"}}, "download_checksums": {"https://zenodo.org/record/7109926/files/train.jsonl": {"num_bytes": 234401262, "checksum": "191992c204ad10d76c2a08005f1cdd94531b531ba12f5ea889cec7cd94dbb232"}, "https://zenodo.org/record/7109926/files/train_mt.jsonl": {"num_bytes": 667946800, "checksum": "f57b8998acba9ac4b06fef7bcb03b8718da7b1d61228efb06afb8088f057d80c"}, "https://zenodo.org/record/7109926/files/val.jsonl": {"num_bytes": 29157311, "checksum": "a395f523de0e536953ac3af7960243b7be90423183097a1856275fc694b6e415"}, "https://zenodo.org/record/7109926/files/test.jsonl": {"num_bytes": 68876958, "checksum": "3e7a6542cd061579599cd93ec4c3af48171f0f8c2331c81477e77fac253247c1"}}, "download_size": 1000382331, "post_processing_size": null, "dataset_size": 922935511, "size_in_bytes": 1923317842}}
```

Comment:
I think you should remove the old configs "all_languages" and "all_languages+mt" from the JSON file.